test/e2e/view-caseload-capacity.js

```diff
@@ -0,0 +1,51 @@
+const expect = require('chai').expect
+
+const workloadCapacityHelper = require('../helpers/data/workload-capacity-helper')
+
+var inserts = []
+
+describe('View your caseload capacity flow', () => {
+  before(function () {
+    return workloadCapacityHelper.selectIdsForWorkloadOwner()
+      .then(function (builtInserts) {
+        inserts = builtInserts
+      })
+  })
+
+  it('should navigate to the workload owner caseload capacity screen', () => {
+    return browser.url('/offender-manager/' + inserts.filter((item) => item.table === 'workload_owner')[0].id + '/caseload-capacity')
+      .waitForExist('.plot-container.plotly')
+      .waitForExist('.heading-large')
+      .getValue('.heading-large', function (title) {
+        expect(title).to.equal('Offender Manager Capacity')
```

Comment:
I expect this will have changed with the new layout to the offender manager's name. A more robust test would be to look for `'Offender Manager'` in the `.heading-secondary` class.